docs/gen/platform-aws-ecs.json

```diff
@@ -483,107 +483,119 @@
          "Optional": false,
          "Default": "",
          "EnvVar": "",
-         "Category": false,
```

Comment:
Can you rebase on `main`? This should be fixed on main.

Reply:
Ohhhh nvm. These are the json diffs from the thing that's on `main`! Carry on.

Reply:
I'm not actually seeing any updates when I rebase onto `main`. :S

Reply:
Yeah you're fine, ignore me! I think somehow the docs updates didn't include the json gen updates when the changes were made that result in you needing to generate some json. This should be fine.

Reply:
should be able to rebase soon, after https://github.com/hashicorp/waypoint/pull/3757 is merged

maybe that will fix the vercel deployment failure, don't know what that's about 🤔